src/Menu/BemMenuWalker.php

```diff
@@ -66,13 +65,18 @@ public function __construct(string $cssClassPrefix)
 	 *
 	 * @return void Parent Display element
 	 */
-	public function display_element($element, &$children_elements, $max_depth, $depth = 0, $args, &$output) // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps, PEAR.Functions.ValidDefaultValue.NotAtEnd
-	{
-
+	public function display_element( // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps, PEAR.Functions.ValidDefaultValue.NotAtEnd
+		$element,
+		&$children_elements,
+		$max_depth, // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps, PEAR.Functions.ValidDefaultValue.NotAtEnd
+		$depth = 0,
+		$args, // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps, PEAR.Functions.ValidDefaultValue.NotAtEnd
+		&$output
+	) {
 		$id_field = $this->db_fields['id'];
 
-		if (is_object($args[0])) {
-			$args[0]->has_children = ! empty($children_elements[ $element->$id_field ]);
+		if (is_object($args[0]) && property_exists($args[0], 'has_children')) {
```

Comment:
Btw you can just do this without raising any warning (it's probably a lot faster):
```suggestion
		if (isset($args[0]-> has_children)) {
```
Source: https://www.php.net/manual/en/function.isset.php#86313

The only difference with isset is that it return FALSE if `has_children == null` but I don't think this is relevant in this case. (source: https://www.php.net/manual/en/function.property-exists.php)

Reply:
Will check what PHPStan says 😄